src/lib/adamantServerApi.js

```diff
@@ -519,6 +520,216 @@ function install (Vue) {
       this.$store.commit('ajax_end_with_error')
     })
   }
+
+  Vue.prototype.getDelegates = function (limit = constants.Delegates.ACTIVE_DELEGATES, offset = 0, votes = []) {
+    this.$store.commit('ajax_start')
+    this.$http.get(this.getAddressString() + `/api/delegates?limit=${limit}&offset=${offset}`).then(response => {
+      if (response.body.success) {
+        for (let i in response.body.delegates) {
+          let delegate = response.body.delegates[i]
+          let voted = votes.includes(delegate.address)
```

Comment:
`let` should be used to the mutable values. If variable value is not going to change, please, use `const` instead.